### ISSUE TITLE
LR and weight decay

### ISSUE THREAD

**Issue by ZN-Qiao** [yitu-opensource/T2T-ViT#25]
Could you please share the learning rate and weight decay used in training? Will they be exactly the same as the commands provided？Is there a recommended setting for batch size 32? Thanks so much!

**Comment by ZN-Qiao** [comment]
Will they be exactly the same as the commands provided？

**Comment by yuanli2333** [comment]
The recommended set of batch size for normal T2T-ViT is 512 (64 in each GPU when 8 GPUs, or 128 for 4 GPUs), and lr=5e-4 for T2T-ViT-14,19,24 and lr=1e-3 for T2T-ViT-7,9,12; all the lr and weight decay has been give in the [training scripts](https://github.com/yitu-opensource/T2T-ViT#train).

**Comment by ZN-Qiao** [comment]
> The recommended set of batch size for normal T2T-ViT is 512 (64 in each GPU when 8 GPUs, or 128 for 4 GPUs), and lr=5e-4 for T2T-ViT-14,19,24 and lr=1e-3 for T2T-ViT-7,9,12; all the lr and weight decay has been give in the [training scripts](https://github.com/yitu-opensource/T2T-ViT#train).

Thanks!
